docs/topics_5/web-create-project.adoc

```diff
@@ -0,0 +1,111 @@
+image::web-custom-labels-project_51.png[Custom labels screen]
++
+You can add custom labels to use in each project by uploading labels or registering a server path that contains a label set.
++
+. If you want to add a label, *Add label*.
+. To upload labels, click *Browse*, select a label file (`\*.mta.label.xml` or `*.windup.label.xml`), and click *Close*.
++
+Uploading labels stores them directly in the {ProductShortName} server.
+
+. To register a server path, click the *Server path* tab, enter the *Labels path* of the label files in the field, and click *Save*.
++
+Registering the server path of a file tells the {ProductShortName} server the location of the labels. This method is useful if you expect to update the labels because it ensures that the {ProductShortName} server always uses the latest version of the labels.
++
+. Click *Next*.
++
+You can customize your analysis by setting some or all of the options on the *Advanced options* screen.
++
+. On the left side of the *Advanced options* screen, complete the following fields:
+
+* *Target*: Select from the list. By default, these are the targets you chose previously.
+* *Source*: Select from the list.
+* *Exclude tags*: Rules with these tags will not be processed. Select from the list.
+* *Additional classpath*: Enter a space-delimited list of additional `.jar` files or directories to add to the class path so that they are available for decompilation or other analysis.
+* *Application name*: Full name of an application.
+* *Mavenize group ID*: Enter the Mavenize group ID.
+* *Ignore path*: Enter a location, in addition to `${user.home}/.mta/ignore/`, for {ProductShortName} to identify files that should be ignored.
+
+. On the right side of the *Advanced options* screen, set the following toggle switches. By default, all are set to disabled.
+
+* *Export CSV*: Exports the report data to a CSV file in your local file system.
+* *Disable Tattletale*: Disables generation of a Tattletale report for each application.
+* *Class Not Found analysis*: Enables analysis of Java files that are not available on the class path. This option should not be used if some classes will be unavailable at analysis time.
+* *Compatible Files report*: Enables generation of the Compatible Files report. This report may take a long time for large applications.
+* *Exploded app*: Indicates that the provided input directory contains source files for a single application.
+* *Keep work dirs*: Instructs {ProductShortName} to not delete temporary working files, such as the graph database and extracted archive files. This option is useful for debugging purposes.
+* *Skip reports*: If enabled, HTML reports are not generated. Enable this toggle if you enabled *Export CSV*.
+* *Allow network access*: Allows network access for features that require it. Currently only validating XML schemas against external resources relies on Internet access. Note that using this option comes with a performance penalty.
```

Comment:
Bit too verbose. Change to:

This option reduces performance.

The first sentence is obvious, so you can remove it. The second is an example and I'm not sure that 'validating XML schemas against external resources' is a task that is documented. If the user is not likely to do this, don't mention it.